extensions/wikia/FlowTracking/FlowTracking.hooks.php

```diff
@@ -1,7 +1,13 @@
 <?php
 
 class FlowTrackingHooks {
-	const CREATE_PAGE_UNRECOGNIZED_FLOW = 'create-page-unrecognized';
+	const FLOWS = [
```

Comment:
Can we do them separately? Using array will hide flow names if you would like to use them in outside class.